manifest_staging/charts/csi-secrets-store-provider-azure/arc-values.yaml

```diff
@@ -0,0 +1,268 @@
+      repository: mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar
+      tag: v2.5.0
+      pullPolicy: IfNotPresent
+    livenessProbeImage:
+      repository: mcr.microsoft.com/oss/kubernetes-csi/livenessprobe
+      tag: v2.6.0
+      pullPolicy: IfNotPresent
+
+  enableSecretRotation: false
+  rotationPollInterval: 2m
+  # Refer to https://secrets-store-csi-driver.sigs.k8s.io/load-tests.html for more details on actions to take before enabling this feature
+  filteredWatchSecret: true
+  
+  syncSecret:
+    enabled: false
+
+  tokenRequests:
+  - audience: api://AzureADTokenExchange
+
+## Install default service account
+rbac:
+  install: true
+  pspEnabled: false
+
+# explicitly reconstruct the pem chain in the order: SERVER, INTERMEDIATE, ROOT
+constructPEMChain: true
+
+# Azure Arc Extension
+enableArcExtension: true
+
+# Values populated by Azure Arc K8s RP during the installation of the extension.
+Azure:
+  Cluster:
+    ResourceId: ""
+    Region: ""
+    Distribution: ""
+    Infrastructure: ""
+  proxySettings:
+    isProxyEnabled: false
+    httpProxy: ""
+    httpsProxy: ""
+    noProxy: ""
+    proxyCert: ""
+  Identity:
+    isEnabled: false
+    Type: ""
+  Extension:
+    Name: ""
+    ResourceId: ""
+
+# Arc monitoring
+arc:
+  enableMonitoring: true
+  promMdmConverter:
+    image:
+      repository: upstreamarc.azurecr.io/prom-mdm-converter
+      tag: v1.0.0
+      pullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 50m
+        memory: 100Mi
+      limits:
+        cpu: 50m
+        memory: 100Mi
+  mdm:
+    image:
+      repository: linuxgeneva-microsoft.azurecr.io/genevamdm
+      tag: master_20220401.1
+      pullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 50m
+        memory: 100Mi
+      limits:
+        cpu: 50m
+        memory: 100Mi
+  msiAdapter:
+    image:
+      repository: mcr.microsoft.com/azurearck8s/msi-adapter
+      tag: 1.0.0
+      pullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 50m
+        memory: 100Mi
+      limits:
+        cpu: 50m
+        memory: 100Mi
+  telegraf:
+    image:
+      repository: mcr.microsoft.com/oss/mirror/docker.io/library/telegraf
+      tag: 1.21
+      pullPolicy: IfNotPresent
+    debugLogging: false
+    resources:
+      requests:
+        cpu: 50m
+        memory: 100Mi
+      limits:
+        cpu: 50m
+        memory: 100Mi
+  amacoreagent:
+    image:
+      repository: pipelineagent.azurecr.io/amacoreagent
+      tag: "3.0"
+      pullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 50m
+        memory: 100Mi
+      limits:
+        cpu: 50m
+        memory: 100Mi
+  fluentd:
+    image:
+      repository: linuxgeneva-microsoft.azurecr.io/genevafluentd_td-agent
+      tag: master_20220403.1
+      pullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 50m
+        memory: 100Mi
+      limits:
+        cpu: 50m
+        memory: 100Mi
```

Comment:
We are treating monitoring as 1p integration. So I think we should not have any aspect of it configurable via help chart. Let me know your thoughts

Reply:
Thanks for the feedback. I have removed these configurations.